primitives/src/v2/mod.rs

```diff
@@ -1569,6 +1594,72 @@ impl CompactStatement {
 	}
 }
 
+/// `IndexedVec` struct indexed by type specific indexes.
```

Comment:
Both `indexes` and `indices` are correct, but we prefer the latter in this codebase
```suggestion
/// `IndexedVec` struct indexed by type specific indices.
```